api/logs/logs.go

```diff
@@ -12,14 +21,18 @@ type SearchRoute struct {
 }
 
 type KubernetesSearchBackend struct {
-	Kubeconfig string `json:"kubeconfig,omitempty"`
+	// empty kubeconfig indicates to use the current kubeconfig for connection
+	Kubeconfig *kommons.EnvVar `json:"kubeconfig,omitempty"`
+	//namespace to search the kommons.EnvVar in
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
The namespace is derived from the search params e.g. labels.namespace or if the id has a `/` the prefix is assumed to be a namespace,

Reply:
But this happening before that. For the request we derive the namespace from the searchParams... but I am not sure how we can derive it for a given config file.
This namespace only indicates which namespace do we need to look for the kubeconfig secret or configMap